packages/elements/src/components/ino-snackbar/ino-snackbar.tsx

```diff
@@ -46,6 +52,10 @@ export class Snackbar implements ComponentInterface {
       this.handleSnackbarHide(e)
     );
     this.snackbarInstance.open();
+    this.snackbarInstance.timeoutMs = -1;
+    if (this.inoTimeout >= 0) {
+      setTimeout(() => this.snackbarInstance.close(), this.inoTimeout);
```

Comment:
Regarding the use of the `setTimeout` function: According to the [mdc documentation](https://github.com/material-components/material-components-web/tree/master/packages/mdc-snackbar), the `timeoutMs` property only supports values between 4000 and 10000 ms.

Reply:
What happens if the consumer provides a value outside that range? If there is no error or warning, we should throw one ourselves and add this information to the description of the property.

Reply:
According to the documentation, an error will be thrown. However, I think we should allow the consumer to set the property to any value and not just to a value within a specific range.